### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 16

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#316] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152323035-8354c4e7-d19c-4ae1-9d46-6ba9498aa816.png)
![image](https://user-images.githubusercontent.com/69389461/152323090-6ba27112-7bb7-416f-bb12-aa8c6cc15bfe.png)

**Comment by Exyss** [comment]
![Esercizio 16](https://user-images.githubusercontent.com/65043207/152660269-12c7c390-b61b-4262-8dbd-00010312b1fe.jpg)
